frontend/public/index.html

```diff
@@ -9,6 +9,10 @@
             rel="stylesheet"
             href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
         />
```

Comment:
Can we add a `<link>` pointing to the `favicon.ico` file?

Reply:
We certainly can but from the sounds of it it might not be needed but i can still add it anyways if you like?